src/EventListener/InitializeSystemListener.php

```diff
@@ -151,6 +167,9 @@ private function boot($routeName, $basePath)
         // Log PHP errors
         $this->iniSet('error_log', $this->rootDir . '/system/logs/error.log');
 
+        // Support legacy Session access
+        $this->initializeLegacySessionAccess();
```

Comment:
I mean accessing the session in `localconfig.php` did not work before, because it was not started?